skins/bancor.scss

```
@@ -1,10 +1,10 @@
 .bancor {
   --primary-color: #10bbd8;
   --bg-color: #18181a;
-  --text-color: #ffffff;
+  --text-color: ffffff;
```

Please do not remove the # for your hexadecimal colour.
```suggestion
  --text-color: #ffffff;
```